src/metabase/pulse/render.clj

```diff
@@ -115,14 +115,14 @@
              (not (#{:waterfall :pie :table :area} display-type)))
         (chart-type :sparkline "result has 2 cols (%s and %s (number)) and > 1 row" (col-description @col-1) (col-description @col-2))
 
-        (= display-type :line)
-        (chart-type display-type "display-type is %s" display-type)
-
         (and (= @col-sample-count 2)
              (number-field? @col-2)
              (= display-type :pie))
         (chart-type :categorical/donut "result has two cols (%s and %s (number))" (col-description @col-1) (col-description @col-2))
 
+        (= display-type :line)
+        (chart-type display-type "display-type is %s" display-type)
+
```

Comment:
I can't figure out why this condition needed to be moved down one. Could you explain it? I noticed it has a similar structure to this condition/result pair: https://github.com/metabase/metabase/pull/25116/files#diff-deeee17ae64e1f3ddb72660dbdd3179ce6a9a10f0bcce127c97af761320804e7R88-R95 I wonder if it would be a good idea to move it into there or not.

Reply:
I'm thinking about the best way to approach this. I actually want to put :line up in the first condition set, since we want to use combo-chart for the line impl too.

But, when I did that, tests were failing. I decided to keep things the same as they were before to try get things merged quickly to unblock some FE stuff.

Long story short, I'm looking at this now

Reply:
In fact, I actually added this line back in in my second commit, and just added it in the wrong place. So this is entirely human error :/